frontends/bmd/src/pages/poll_worker_screen.tsx

```diff
@@ -172,65 +164,123 @@ function PrecinctScannerTallyReportModal({
+              <p>
+                The polls are now open. If needed, you may print additional
+                copies of the polls opened report.
+              </p>
+            </React.Fragment>
+          ) : (
+            <React.Fragment>
+              <h1>Polls Closed Report Printed</h1>
+              <p>
+                The polls are now closed. If needed, you may print additional
+                copies of the polls closed report.
+              </p>
+            </React.Fragment>
+          )}
+        </Prose>
+      );
+      modalActions = (
+        <React.Fragment>
+          <Button primary onPress={onClose}>
+            Continue
+          </Button>
+          <Button onPress={printAdditionalReport}>
+            Print Additional Report
+          </Button>
+        </React.Fragment>
+      );
+      break;
+    /* istanbul ignore next - compile time check for completeness */
+    default:
+      throwIllegalValue(modalState);
+  }
+
   return (
     <React.Fragment>
-      {!isPrinting && (
-        <Modal
-          content={
-            <Prose id="modalaudiofocus">
-              {precinctScannerTally.isPollsOpen ? (
-                <React.Fragment>
-                  <h1>Polls Opened Report on Card</h1>
-                  <p>
-                    This poll worker card contains a polls opened report. After
-                    printing, the report will be cleared from the card and the
-                    polls will be opened on VxMark.
-                  </p>
-                </React.Fragment>
-              ) : (
-                <React.Fragment>
-                  <h1>Polls Closed Report on Card</h1>
-                  <p>
-                    This poll worker card contains a polls closed report. After
-                    printing, the report will be cleared from the card and the
-                    polls will be closed on VxMark.
-                  </p>
-                </React.Fragment>
-              )}
-            </Prose>
-          }
-          actions={
-            <Button primary onPress={printReportAndTogglePolls}>
-              {precinctScannerTally.isPollsOpen
-                ? 'Open Polls and Print Report'
-                : 'Close Polls and Print Report'}
-            </Button>
-          }
-        />
-      )}
-      {isPrinting && (
-        <Modal
-          content={
-            <Prose textCenter id="modalaudiofocus">
-              <Loading>
-                {precinctScannerTally.isPollsOpen
-                  ? 'Printing polls opened report'
-                  : 'Printing polls closed report'}
-              </Loading>
-            </Prose>
-          }
-        />
-      )}
+      {/* 
+        This modal is torn down in some way which react-modal doesn't expect
+        and as a result, can leave the app hidden (in terms of aria). 
+        https://votingworks.slack.com/archives/CEL6D3GAD/p1633723756081200 
+      */}
+      <Modal
+        ariaHideApp={false}
```

Comment:
My changes to this page somehow reintroduced the issue where the accessibility controller (or the arrow keys) would not longer work to navigate on VxMark after this modal was opened.

This `ariaHideApp={false}` was added as a result of the original issue (https://votingworks.slack.com/archives/CEL6D3GAD/p1633723756081200), but at some point was removed. I tried experimenting with different permutations of the modal - but based on @eventualbuddha's comments in that thread, it may be that we're rarely closing modals gracefully and in this unique case react-modal can't handle it? In any case, this fix works and we are not prioritizing accessibility on the poll worker screens right now.

Props to better RTL practices, like using `userEvent` and `getByRole`, which caught this in automated testing!

Reply:
cc @benadida

Reply:
@benadida sounds good, I created a ticket to track (#2618) and link to that instead.

@arsalansufi, I'm not sure that's an acceptable comment format for JSX? I tried it and the linter didn't like it and the code wouldn't run. I added an extra line to specify we're talking about that prop though.

Reply:
Ah! You probably have to use the standard backslashes when commenting a prop:

```jsx
<Modal
  // This modal is torn down in a way that react-modal doesn't expect and, as
  // a result, can leave the app hidden (in terms of accessibility). We set
  // this prop to false to disable the app hiding altogether
  ariaHideApp={false}
```